packages/cli/src/utils/changelog.js

```diff
@@ -34,16 +71,34 @@ const getChangelogFromMarkdown = (version, markdown) => {
   endingLine -= 1;
 
   const changelogLines = lines.slice(startingLine, endingLine);
+  const changelog = [];
+  const appMetadata = [];
+  const issueMetadata = [];
+
+  for (const line of changelogLines) {
+    if (line.startsWith('APP')) {
+      appMetadata.push(...getAppMetadata(line));
+    } else if (line.startsWith('ISSUES')) {
+      issueMetadata.push(...getIssueMetadata(line));
+    } else {
+      // We skip lines that contain metadata as they will be submitted separately
+      changelog.push(line);
+    }
+  }
 
-  return changelogLines.join('\n');
+  return {
+    changelog: changelog.join('\n'),
+    appMetadata: absentIfEmpty(appMetadata),
+    issueMetadata: absentIfEmpty(issueMetadata),
+  };
 };
 
-const getVersionChangelog = (version, appDir = '.') => {
+const getVersionChangelog = async (version, appDir = '.') => {
```

Comment:
Why do we want to make this call async?

Reply:
It was already async because it uses a Promise and returns the result. This makes that more clear/impossible to misuse as a caller because it'll mandate an `await` vs. having to know you should use one previously.